docs/book/src/reference/versions.md

```diff
@@ -67,9 +67,10 @@ The table below documents support matrix for Cluster API versions (versions olde
 
 | Minor Release | Status                  | Supported Until (including maintenance mode)                                                |
 |---------------|-------------------------|---------------------------------------------------------------------------------------------|
+| v1.12.x       | Standard support period | in maintenance mode when v1.14.0 will be released, EOL when v1.15.0 will be released        |
 | v1.11.x       | Standard support period | in maintenance mode when v1.13.0 will be released, EOL when v1.14.0 will be released        |
-| v1.10.x       | Standard support period | in maintenance mode when v1.12.0 will be released, EOL when v1.13.0 will be released        |
-| v1.9.x        | Maintenance mode        | Maintenance mode since 2025-08-12 - v1.11.0 release date, EOL when v1.12.0 will be released |
+| v1.10.x       | Maintenance mode        | Maintenance mode since 2025-12-18 - v1.12.0 release date,, EOL when v1.13.0 will be released  |
```

Comment:
```suggestion
| v1.10.x       | Maintenance mode        | Maintenance mode since 2025-12-18 - v1.12.0 release date, EOL when v1.13.0 will be released  |
```